Decred Wallet/view_util/DropMenuButton.swift

```diff
@@ -104,22 +131,35 @@ class DropMenuButton: UIButton, UITableViewDelegate, UITableViewDataSource
 
         // set automatically the selected item index to 0
         self.selectedItemIndex = 0
-        self.selectedItem = self.items[self.selectedItemIndex]
+        self.selectedItem = self.items[self.selectedItemIndex].text
         self.setTitle(self.selectedItem!, for: .normal)
         self.setTitle(self.selectedItem!, for: .selected)
         self.setTitle(self.selectedItem!, for: .highlighted)
 
         table.reloadData()
     }
     
+    func caculateWidthCell() {
+        self.items.forEach { (item) in
+            let widthText = item.text.width(withConstrainedHeight: 20, font: UIFont(name: "SourceSansPro-Regular", size: 17) ?? UIFont.systemFont(ofSize: 17))
```

Comment:
value of type string has no member width

Reply:
Of course it have

Reply:
I get this error when I try to compile `Value of type 'String' has no member 'width'`. @JustinBeBoy confirm that you have the current implementation.